src/core/blockchain/near/config.ts

```diff
@@ -51,6 +87,29 @@ export const config: IBlockchainConfig = {
         }
     },
     ui: {
+        validator: {
+            totalLabel: 'Validator.totalStakes',
+            amountCardLabel: 'App.labels.staked',
+            maximumNumberOfValidators: 5
```

Comment:
If we don’t know exactly the number or there is no limit remove it - its optional